test/RecipeRepository-test.js

```diff
@@ -28,11 +28,22 @@ describe('RecipeRepo', () => {
     expect(recipeRepository.createRecipes(sampleRecipeData)).to.deep.equal([recipe1, recipe2])
   })
   it('Should filter a list of recipes based on a tag.', () => {
-    expect(recipeRepository.filterTag("lunch")).to.deep.equal([recipe2])
-    expect(recipeRepository.filterTag("hor d'oeuvre")).to.deep.equal([recipe1])
+    expect(recipeRepository.filterTag('lunch')).to.deep.equal([recipe2])
+    expect(recipeRepository.filterTag('hor d\'oeuvre')).to.deep.equal([recipe1])
+  })
+  it('Should return an empty array if the tag passed in does not match any recipes.', () => {
+    expect(recipeRepository.filterTag('take-out')).to.deep.equal([])
+    expect(recipeRepository.filterTag('food')).to.deep.equal([])
   })
   it('Should filter list of recipes based on its name.', () => {
-    expect(recipeRepository.filterName("maple dijon apple cider grilled pork chops")).to.deep.equal([recipe2])
-    expect(recipeRepository.filterName("loaded chocolate chip pudding cookie cups")).to.deep.equal([recipe1])
+    expect(recipeRepository.filterName('maple dijon apple cider grilled pork chops')).to.deep.equal([recipe2])
+    expect(recipeRepository.filterName('loaded chocolate chip pudding cookie cups')).to.deep.equal([recipe1])
+  })
+  it('Should return an empty array if the name searched does not match any recipes.', () => {
+    expect(recipeRepository.filterName('calamari')).to.deep.equal([])
+    expect(recipeRepository.filterName('hamburger')).to.deep.equal([])
+  })
+  it('Should return all recipes if an empty string is passed in.', () => {
+    expect(recipeRepository.filterName('')).to.deep.equal([recipe1, recipe2])
```

Comment:
Hi Eleanor, the code I just implemented for the search bar error message says that if the user puts in an empty string, the error message will pop up. Should we change this to give an empty array or should I go back and change the code?